src/app/app.component.html

```diff
@@ -1,49 +1,126 @@
-<div class="wrapper  w-full flex justify-center">
+<div class="wrapper w-full flex justify-center">
   <div class="w-full max-w-[1440px]">
-    <nav class="bg-transparent w-full z-20 text-black flex justify-between items-center h-[90px] max-w-[1200px] mx-auto mb-[57px]">
-      <div class="h-[70px] justify-between items-center w-full flex">
+    <nav
+      class="w-full bg-[#FFFFFF] fixed md:static z-20 text-black flex justify-between items-center h-[90px] max-w-[1200px] mx-auto lg:mb-[57px] md:mb-0 px-5 md:px-0">
+      <div class="h-[60px] md:h-[70px] justify-between items-center w-full flex">
         <div class="flex items-center">
-          <a href="#">
-            <img src="assets/new_logo.svg" alt="Bee O`clock Logo">
+          <a href="#" class="flex gap-[14px]">
+            <img ngSrc="assets/new_logo.svg" alt="Bee O`clock Logo" height="38" width="38">
+            <img class="hidden md:block" ngSrc="assets/logo_text.svg" alt="Bee O`clock" height="16" width="119">
           </a>
         </div>
+        <div>
+          <img class="md:hidden block" ngSrc="assets/logo_text.svg" alt="Bee O`clock" height="16" width="119">
+        </div>
+        <div class="md:hidden">
+          <button (click)="toggleMobileMenu()" class="text-black focus:outline-none h-[100%]">
```

Comment:
Please add `id` attribute for each action element

Reply:
Also add please the class `text-2xl`, or find much better size, we should do much bigger the icons because they so small and hard to click

Reply:
done